backend/src/eml/base.rs

```diff
@@ -0,0 +1,136 @@
+use std::io::BufRead;
+
+use quick_xml::{
+    se::{Serializer, WriteResult},
+    DeError, SeError,
+};
+use serde::{de::DeserializeOwned, Deserialize, Serialize};
+
+/// Base EML XML document that contains all the mostly irrelevant for our logic
+/// XML tags and setup.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct EMLBase {
+    #[serde(rename = "@Id")]
+    pub id: String,
+    #[serde(rename = "@SchemaVersion")]
+    pub schema_version: String,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub canonicalization_method: Option<CanonicalizationMethod>,
```

Comment:
`CanonicalizationMethod` is part of the XML Signature (`http://www.w3.org/2000/09/xmldsig#`) namespace. But because we're not using XML signatures, I think we should remove that namespace and the `CanonicalizationMethod` element entirely.

Reply:
I agree, I've removed it in [1d7cb0f](https://github.com/kiesraad/abacus/pull/637/commits/1d7cb0fb6d9282ba8d495494290d4e330bf5b166)